### PR TITLE
Move social icons to nav panel on mobile

### PR DESCRIPTION
**Pull request by AndrewLester**

### CODE REVIEW

File: js/header.js [hunk 1]
```diff
@@ -1,9 +1,37 @@
-document.onscroll = function() {shrinkHeader()};
 header = document.getElementsByTagName('header')[0];
+
 function shrinkHeader() {
-    if (document.body.scrollTop > 50 && document.body.clientWidth >= 1200 || document.documentElement.scrollTop > 50 && document.body.clientWidth >= 1200) {
+    if ((document.documentElement.scrollTop || document.body.scrollTop) > 50 || document.body.clientWidth <= 1275) {
```
Comment (ErikBoesen) [inline js/header.js:4]: What's the reason for this change?

File: js/header.js [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
         header.className = 'shrink';
-    } else if (document.body.clientWidth >= 1200){ 
+    } else {
         header.className = '';
     }
 }
+
+//Shrink header on mobile and resized window
```
Comment (ErikBoesen) [inline js/header.js:11]: Space

File: js/header.js [hunk 3]
```diff
@@ -1,9 +1,37 @@
...
+body.onload = function () {
+    if (body.clientWidth <= 1275) {
+        header.className = 'shrink';
+    } else {
+        header.className = '';
+    }
+};
+body.onresize = function () {
+    if (body.clientWidth <= 1275) {
```
Comment (ErikBoesen) [inline js/header.js:20]: Not necessarily a problem for right now, but might be good to declare some constants instead of all these magic numbers

**Review by ErikBoesen — APPROVED** [review]

File: js/header.js (hunk 1 above)
Comment (AndrewLester) [reply js/header.js:4]: I thought `documentElement` was too new but it seems pretty widely available. `document.body.scrollTop` kept giving back 0 scroll.

Comment (ErikBoesen) [reply js/header.js:4]: Weird. It's definitely worked before.

Comment (AndrewLester) [reply js/header.js]: My chrome may have recently updated or something. It seems like `document.body.scrollTop` got deprecated at some point.
